docs/introduction/vulkan_overview.md

```diff
@@ -9,57 +9,61 @@ parent: introduction
 
-During this guide we will explain the things that are most different between the 2 types of GPUs.
+During this guide we will explain many of the differences between the two types of GPUs.
 
 ## Validation layers
 
-There is a significant amount of things in the Vulkan API that you have to do, but they actually do nothing on the real hardware. This means that even on some wrong code, it can still run just fine on some hardware, but break spectacularly on others. This is specially bad on syncronization code, which is very hard to get right, and it can run very different depending on the hardware unless you do everything according to the spec. Something that runs fine on a Nvidia card might break on an AMD card, and its the same in reverse.
+The Vulkan API is very large and rather easy to make mistakes with. Fortunately, when Vulkan was being designed, the creators realized that and made the Validation Layers to fix it. The Validation Layers is a bunch of error checking code that detects and reports 'invalid usage' of the API. They check for all kinds of errors such as using the wrong enums, threading violations, and object lifetimes. The layers are separate from the driver, and to work, they must intercept every function call, perform validation, then call the driver for you. The reason for that is when you want to turn off validation, you Vulkan calls go directly to the driver and can run as fast as possible. 
+
+While developing a Vulkan application, you should have the Validation Layers enabled. The obvious reason is that they detect bugs, but another big reason is that by having a clean validation output you can reasonably assume your application will work across a wide variety of hardware. This is especially important for synchronization, which is tricky to get right, and what may work perfectly fine on one GPU may fail spectacularly on another GPU.
 
-Those are some of the main reasons Vulkan comes with a set of "Validation Layers". They are hooked between the API calls from your code, and the actual gpu driver, and they check that you are doing the correct things according to the Vulkan spec. While developing, its imperative to use them to make sure you arent doing the wrong things. They can be enabled and disabled at startup, so when profiling or releasing the aplication, you would usually disable them. They do bring a significant hit to performance due to all that checking. 
+It is important to note that the Validation Layers can't catch every possible bug, like as uninitialized memory and bad pointers. There are tools that can check for these bugs, such as Address Sanitizer and valgrind, and using them is highly advised. Unfortunately, graphics drivers often create false positives while using these tools, making their output quite noisy. 
```

Comment:
`like as uninitialized memory`? isnt this sentence weird?

Reply:
Yes, as shouldn't be there. Will fix it in a bit.

Reply:
Fixed in the forced push.